assets/javascripts/discourse/components/upcoming-events-list.gjs

```diff
@@ -161,35 +162,40 @@ export default class UpcomingEventsList extends Component {
           {{/if}}
 
           {{#unless this.isLoading}}
-            {{#each-in this.eventsByMonth as |month monthData|}}
-              {{#if this.monthFormat}}
-                <h4 class="upcoming-events-list__formatted-month">
-                  {{this.formatMonth month}}
-                </h4>
-              {{/if}}
-
-              {{#each-in monthData as |day events|}}
-                <div class="upcoming-events-list__day-section">
-                  <div class="upcoming-events-list__formatted-day">
-                    {{this.formatDate month day}}
+            <PluginOutlet
+              @name="upcoming-events-list-container"
+              @connector_tag_name="div"
```

Comment:
```suggestion
```
This isn't a supported argument for PluginOutlet

Reply:
done! thank you